tests/modules/users/resources/test_getting_users_info.py

```diff
@@ -111,3 +112,49 @@ def test_getting_user_id_not_found(flask_app_client, regular_user):
     ):
         response = flask_app_client.get('/api/v1/users/wrong-uuid')
         assert response.status_code == 404
+
+
+def test_getting_sightings_for_user(flask_app_client, staff_user, db):
+
+    from tests.modules.sightings.resources import utils as sighting_utils
+
+    temp_owner = utils.generate_user_instance(
+        email='user_4_sightings@localhost',
+        is_researcher=True,
+    )
+
+    data_in = {
+        'encounters': [{}],
+        'context': 'test',
+        'locationId': 'test',
+    }
+
+    sighting_create_response = sighting_utils.create_sighting(
+        flask_app_client, temp_owner, expected_status_code=200, data_in=data_in
+    )
+
+    from app.modules.sightings.models import Sighting
+
+    sighting_id = sighting_create_response.json['result']['id']
+    sighting = Sighting.query.get(sighting_id)
+
+    assert sighting_create_response.json['success']
+    assert sighting is not None
+    assert sighting.encounters[0].owner is temp_owner
+    assert str(temp_owner.get_sightings()[0].guid) == sighting_id
+
+    with flask_app_client.login(temp_owner, auth_scopes=('users:read',)):
```

Comment:
re comment about permissions, prove if I'm correct by trying to read this as a different user.